src/routes/userRoutes.js

```diff
@@ -164,13 +164,13 @@ routes.createDatabase = async (req, res) => {
   }
 
   try {
-    if (req.params.database === "postgres") {
+    if (req.params.database === "Postgres") {
       await pgModule.createPgAccount(username, dbPassword);
       return res
         .status(200)
```

Comment:
You didn't write this, but can you fix this? The `.status(200)` is unneeded because the status defaults to 200.

Reply:
Also on line 175 too :)